### ISSUE TITLE
The .pc files inside RPM packages contain incorrect 'prefix', pointing to '/vagrant-substrate'

### ISSUE THREAD

**Issue by jhernand** [mitchellh/vagrant#8184]
### Vagrant version

Vagrant 1.9.1, installed using the CentOS RPM:

```
# rpm -q 
vagrant-1.9.1-1.x86_64
```

### Host operating system

CentOS 7.

### Guest operating system

Not relevant.

### Vagrantfile

Not relevant.

### Debug output

See the references.

### Expected behavior

The `.pc` files included in the Vagrant packages should contain a `prefix` variable pointing to the actual location where they are installed. For example, for the CentOS 7 RPM packages it should be '/opt/vagrant/embedded`.

### Actual behavior

The `prefix` variable points to a directory that isn't part of the RPM installation. For eample, for the `libxml2` package:

```
# grep '^prefix=' /opt/vagrant/embedded/lib/pkgconfig/libxml-2.0.pc 
prefix=/vagrant-substrate/staging/embedded
...
```

As you see the `prefix` points to `/vagrant-substrate/staging/embedded`, which isn't installed by the RPM package.

The same happens with all the other `.pc` files, and with the tools like `xml2-config` and `curl-config`:

```
# /opt/vagrant/embedded/bin/xml2-config --cflags
-I/vagrant-substrate/staging/embedded/include/libxml2
```

### Steps to reproduce

1. Install the Vagrant RPM.
2. Check the `.pc` files.

### References

This issue was discovered initially as part of a bug related to the oVirt SDK: https://bugzilla.redhat.com/1411594

**Comment by myoung34** [comment]
FWIW the relevant code is [here](https://github.com/mitchellh/vagrant-installers/blob/master/substrate/modules/libxslt/manifests/init.pp)

**Comment by chrisroberts** [comment]
Fixed via mitchellh/vagrant-installers#100

**Comment by jhernand** [comment]
@chrisroberts @mitchellh Thanks!
